ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ethernet-interfaces.md

```diff
@@ -162,14 +165,27 @@ sFlow is disabled.
 
 | Interface | PVLAN Mapping | Secondary Trunk |
 | --------- | ------------- | ----------------|
+| Ethernet1 | 20-30 | True |
+| Ethernet2 | - | False |
 | Ethernet15 | 111 | - |
 | Ethernet17 | - | True |
 
 ##### VLAN Translations
 
-| Interface | From VLAN ID(s) | To VLAN ID | Direction |
-| --------- | --------------- | -----------| --------- |
-| Ethernet16 | 111-112 | 110 | out |
+| Interface |  Direction | From VLAN ID(s) | To VLAN ID | From Inner VLAN ID | To Inner VLAN ID | Network | Dot1q-tunnel |
```

Comment:
There is a space too much in there. Also for some of the headings here they should be matched with the Flexible Encapsulation section in line 155 of this file.

Reply:
Fixed